File: java64/nivel1/exer9.java
import javax.swing.JOptionPane;

public class exer9 {
    public static void main(String[] args) {
        int ano, mes, dia, idade;

        ano = Integer.parseInt(JOptionPane.showInputDialog(null, "Idade anos: "));
        mes = Integer.parseInt(JOptionPane.showInputDialog(null, "Idade meses: "));
        dia = Integer.parseInt(JOptionPane.showInputDialog(null, "Idade dias: "));

        idade = (365 * ano) + (30 * mes) + dia;

        JOptionPane.showMessageDialog(null, "Idade em dias: " + idade);
    }
}

File: pyhton64/nivel1/exer9.py
idade_ano = int(input('IDade em anos: '))
idade_mes = int(input('Idade em meses: '))
idade_dia = int(input('Idade em dias: '))

idade = (idade_ano * 365) + (idade_mes * 30) + idade_dia
print(f'Idade: {idade}')